std/conv.d

```diff
@@ -3980,10 +3982,14 @@ private template emplaceImpl(T)
                 else static if (is(Unqual!(ElementEncodingType!Arg) == UE))
                 {
                     import core.stdc.string : memcpy;
-                    assert(N == chunk.length, "Array length missmatch in emplace");
-                    memcpy(cast(void*)&chunk, arg.ptr, T.sizeof);
+                    assert(n == chunk.length, "Array length missmatch in emplace");
+
+                    // This is unsafe as long as the length match is a
+                    // precondition and not an unconditional exception
+                    memcpy(&chunk, arg.ptr, T.sizeof);
+
                     static if (hasElaborateCopyConstructor!T)
-                        typeid(T).postblit(cast(void*)&chunk);
+                        _postblitRecurse(chunk);
```

Comment:
Is it a problem if it fails? As of a recent patch to DMD, `typeid(...).postblit` handles failed postblits properly, and `_postblitRecurse` handles it too.